packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -0,0 +1,35 @@
+# Default values for rancher-cis-benchmark.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  cisoperator:
+    repository: rancher/cis-operator
+    tag: v0.0.2
+  securityScan:
+    repository: prachidamle/security-scan
```

Comment:
hmmm, this is not pushed under `rancher`

Reply:
Need this PR merged for building the rancher/security-scan image https://github.com/rancher/security-scan/pull/23

Reply:
Need to get this in ASAP as it will block airgap testing